client/ayon_core/pipeline/tempdir.py

```diff
@@ -3,11 +3,80 @@
 """
 
 import os
+import tempfile
+from pathlib import Path
 from ayon_core.lib import StringTemplate
 from ayon_core.pipeline import Anatomy
 
 
-def create_custom_tempdir(project_name, anatomy=None):
+def get_temp_dir(
+    project_name=None, anatomy=None, prefix=None, suffix=None, make_local=False
+):
+    """Get temporary dir path.
+
+    If `make_local` is set, tempdir will be created in local tempdir.
```

Comment:
All these additional information to arguments should be next to arguments definitions below. BTW not sure what `make_local` does? That might need better variable name.